arenadata/Dockerfile.ubuntu

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:22.04 AS base
+
+COPY README.Ubuntu.bash ./
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -eux; \
+    ./README.Ubuntu.bash; \
+    rm README.Ubuntu.bash; \
+# The en_US.UTF-8 locale is needed to run GPDB
+    locale-gen en_US.UTF-8; \
+# To run sshd directly, but not using `/etc/init.d/ssh start`
+    mkdir /run/sshd; \
+# Alter precedence in favor of IPv4 during resolving
+    echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
+# Packages for tests
+    apt install -y rsync iproute2 krb5-kdc krb5-admin-server fakeroot lsof sudo python3.11 python3.11-dev python3-pip openjdk-11-jdk libipc-run-perl;\
```

Comment:
Some of these packages are already installed by the README.Ubuntu.bash script above.